modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1052,38 +1058,54 @@ function DOMUtils(doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
 
   const createRng = () => doc.createRange();
 
-  const split = (parentElm: Node, splitElm: Node, replacementElm?: Node) => {
-    let r = createRng(), bef, aft, pa;
+  const split = (parentElm: Node, splitElement: Node, replacementElement?: Node, optionsInput?: SplitOptions) => {
```

Comment:
This type definition doesn't line up with the ones above

Reply:
As if a function needs to line up with the type...